urth-viz-explorer.html

```diff
@@ -470,7 +475,7 @@
                         dataframe.limit = event.detail.value;
                     }.bind(this));
                     dataframe.addEventListener('value-changed', function(event) {
```

Comment:
suggestion: `dataframe.addEventListener('value-changed', this._dataChanged.bind(this))`